mcserver/settings.py

```diff
@@ -275,6 +275,10 @@ def strip_sensitive_data(event, hint):
         'task': 'mcserver.tasks.delete_pingdom_sessions',
         'schedule': crontab(hour='*', minute=0)
     },
+    'submit_cloudwatch_metric': {
+        'task': 'mcserver.tasks.submit_cloudwatch_metrics',
+        'schedule': crontab(minute='*')
```

Comment:
@olehkorkh-planeks @sashasimkin  @suhlrich is it possible to have a finer resolution or is 1 minute the best we can do? It seems a little slow

Reply:
@antoinefalisse this is what I suggested [here in point #1](https://github.com/stanfordnmbl/opencap-api/issues/173#issuecomment-2093012528).

Reply:
Can you explain what you mean with aggregates of 1 minutes?

Reply:
@antoinefalisse check [this documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-scaling-target-tracking.html#:~:text=Target%20tracking%20will%20evaluate%20metrics%20aggregated%20at%20a%20one%2Dminute%20granularity%20for%20all%20predefined%20metrics%20and%20custom%20metrics) about target tracking(search for "aggregated at a one-minute granularity" text if highlight doesn't work).

Few more arguments against higher resolution metrics:
1. From what I understand the trials processed around 1-60 minutes, so I don't expect rapid changes when all trials are processed
2. I've configured scale in/out cooldowns to 5m to avoid instances flapping
3. I saw in the code logic to cool-down the worker instance before unprotecting it, which adds to the delay